packages/xrpl/src/utils/index.ts

```diff
@@ -58,6 +58,7 @@ import {
 import signPaymentChannelClaim from './signPaymentChannelClaim'
 import { convertHexToString, convertStringToHex } from './stringConversion'
 import {
+  getCurrentUnixTimestamp,
```

Comment:
Is it more useful to have a helper function to get the current timestamp or just to convert to/from normal UNIX time (millisecond time)?

Reply:
It's better to just get the current timestamp because there's no XRPL use case to convert from/to ms/seconds.

Reply:
There is now, right? Due to this feature?

Reply:
I meant other transactions don't require unix timestamps in milliseconds.

Reply:
But I also do see how it would be convenient for app developers to pass in their already initiated `Date.now()` millisecond timestamp into a helper function that would convert it to seconds.

Reply:
That's fair.

Reply:
After further thought, it's no longer necessary. Removed in https://github.com/XRPLF/xrpl.js/pull/2688/commits/80d91eca3caf7852db1c685e84394c139c835bf4